next-unbuilt-package.sh

```diff
@@ -48,10 +49,14 @@ for CRATE in $POPULAR_CRATES; do
     continue
   fi
 
-  rm -rf "$TEMPDIR/crates.io-response.json"
-  curl_slowly --location --fail "https://crates.io/api/v1/crates/${CRATE}" >"$TEMPDIR/crates.io-response.json"
-  VERSION=$(cat "$TEMPDIR/crates.io-response.json" | jq -r .versions[0].num)
-  LICENSE=$(cat "$TEMPDIR/crates.io-response.json" | jq -r .versions[0].license | sed -e 's:/:", ":g' -e 's/ OR /", "/g')
+  RESPONSE_DIR="$TEMPDIR/crates.io-responses/"
+  RESPONSE_FILENAME="$RESPONSE_DIR/$CRATE.json"
+  if [[ ! -f "$RESPONSE_FILENAME" ]]; then
+    mkdir -p "$RESPONSE_DIR"
+    curl_slowly --location --fail "https://crates.io/api/v1/crates/${CRATE}" >"$RESPONSE_FILENAME"
+  fi
+  VERSION=$(cat "$RESPONSE_FILENAME" | jq -r .versions[0].num)
+  LICENSE=$(cat "$RESPONSE_FILENAME" | jq -r .versions[0].license | sed -e 's:/:", ":g' -e 's/ OR /", "/g')
```

Comment:
```suggestion
  LICENSE=$(jq -r '.versions[0].license' "$RESPONSE_FILENAME" | sed -e 's:/:", ":g' -e 's/ OR /", "/g')
```

Reply:
Due to some internal github error, I created multiple duplicate reviews, closing this as duplicate.